content/blog/2017/2017-02-06-gsoc2017-announcement.adoc

```diff
@@ -0,0 +1,72 @@
+---
+:layout: post
+:title: "Jenkins Google Summer Of Code 2017. Call for mentors"
+:tags:
+- gsoc
+- events
+- general
+- gsoc2017
+:author: oleg-nenashev
+---
+
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/oleg-nenashev[Oleg Nenashev],
+one of the Jenkins Organization Admins in Google Summer of Code.
+====
+
+On behalf of the GSoC org team I am happy to announce that we are going to apply to 
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code (GSoC) on this year]. 
+GSoC presumes that high-profile students work in open-source projects for several months under mentorship of organization members.
+
+We are looking for mentors and project ideas. 
+So yes, we are looking for you :)
+
+== Conditions
+
+* Mentors lead the project in the area of their interest
+* Mentors actively participate in the project during student selection, community bonding and coding phases (March - August)
+* Mentors work in teams of 2+ mentors per 1 each student
+* Mentors dedicate a consistent and significant amount of time, especially during the coding phase (_~5 hours_ per week in the team of two mentors)
+
+Mentorship does not require strong expertise in Jenkins plugin development. 
+The main objective is to guide students and to get them involved into the Jenkins community. 
+If your mentor team requires any specific expertise, org admins will do their best in order to find advisors.
+
+== What do you get?
+
+* A student, who works within the area of your interest on full-time for several months
+* Joint projects with Jenkins experts, lots of fun and ability to study something together
+* Limited edition of swags from Google and Jenkins project
+* Maybe: Participation in GSoC Mentor Summit in California with expense coverage (depends on project results and per-project quotas)
+
+== Requirements
+
+* You are passionate about Jenkins
+* You are interested in being mentor or advisor
+* Your project idea should be about Jenkins (plugins, core, infrastructure, etc.)
+* Your project idea should be about code
+* Your project idea should be potentially doable in 3-4 months
+* You are ready to dedicate time && have no major unavailability periods planned to this summer 
+
+== If you are interested
+
+In such case drop the Email to the Jenkins Developer mailing list with the _GSoC2017_ prefix.
+
+* Briefly describe your project idea (several sentences) and required qualifications from students. link:https://wiki.jenkins-ci.org/display/JENKINS/Google+Summer+Of+Code+2016#GoogleSummerOfCode2016-Projectideas[Examples from GSoC2016]
```

Comment:
"a couple of sentences" would probably sound better and not sound too cumbersome to produce.